cpp/bin/torchtrtc/README.md

```diff
@@ -14,112 +14,115 @@ to standard TorchScript. Load with `torch.jit.load()` and run like you would run
-                                      64bit to 32bit (Long, Double to Int,
-                                      Float)
-    --save-engine                     Instead of compiling a full a
-                                      TorchScript program, save the created
-                                      engine to the path specified as the
-                                      output path
-    input_file_path                   Path to input TorchScript file
-    output_file_path                  Path for compiled TorchScript (or
-                                      TensorRT engine) file
-    input_specs...                    Specs for inputs to engine, can either
-                                      be a single size or a range defined by
-                                      Min, Optimal, Max sizes, e.g.
-                                      "(N,..,C,H,W)"
-                                      "[(MIN_N,..,MIN_C,MIN_H,MIN_W);(OPT_N,..,OPT_C,OPT_H,OPT_W);(MAX_N,..,MAX_C,MAX_H,MAX_W)]".
-                                      Data Type and format can be specified by
-                                      adding an "@" followed by dtype and "%"
-                                      followed by format to the end of the
-                                      shape spec. e.g. "(3, 3, 32,
-                                      32)@f16%NHWC"
-    "--" can be used to terminate flag options and force all following
-    arguments to be treated as positional options
+      -h, --help                        Display this help menu
+      Verbiosity of the compiler
+        -v, --verbose                     Dumps debugging information about the
+                                          compilation process onto the console
+        -w, --warnings                    Disables warnings generated during
+                                          compilation onto the console (warnings
+                                          are on by default)
+        --i, --info                       Dumps info messages generated during
+                                          compilation onto the console
+      --build-debuggable-engine         Creates a debuggable engine
+      --use-strict-types                Restrict operating type to only use set
+                                        operation precision
+      --allow-gpu-fallback              (Only used when targeting DLA
+                                        (device-type)) Lets engine run layers on
+                                        GPU if they are not supported on DLA
+      --require-full-compilation        Require that the model should be fully
+                                        compiled to TensorRT or throw an error
+      --is-supported=[check-method-op-support],
+      --supported=[check-method-op-support],
+      --check-support=[check-method-op-support],
+      --check-method-op-support=[check-method-op-support]
```

Comment:
I wanted to have a few short cuts so that someone should be able to just guess the right one. If we can choose a good single option I am okay with simplifying

Reply:
I'm slightly inclined towards `--check-support`

Reply:
how about `--check-method-support`? since you need the argument

Reply:
sounds good !!